docs/astro/src/content/docs/reference/overview.mdx

```diff
@@ -99,18 +99,24 @@ Currently the `z` value is a compile time constant and cannot be changed at runt
 </SlintProperty>
 
 ### absolute-position
-<SlintProperty propName="absolute-position" typeName="struct" structName="Point" > 
-The position of the element within the contained window.
+<SlintProperty propName="absolute-position" typeName="struct" structName="Point" propertyVisibility="out">
+
+Represents a point that specifies an absolute position within a window or popup window.
+It defines coordinates (x,y) relative to the enclosing Window or PopupWindow, but the reference frame is unspecified
+(could be screen, window, or popup coordinates).
+This property is only absolute within the enclosing Window or PopupWindow.
+To use correctly, typically needs to be subtracted from another absolute-position (e.g. from another item from the same window).
```

Comment:
This sentence is really weird. It starts optimistic with "to use correctly", then softens with "typically" and then leaves the reader hanging with "another". And the sentence is missing a subject.

I suggest to either rephrase it with a concrete example or leave it out.

Similarly the previous sentence I'd also leave out for being redundant.